examples/hf_trainer_api/image_classification/requirements.txt

```diff
@@ -4,5 +4,5 @@ torch>=1.5.0
 torchvision>=0.6.0
 datasets
 evaluate
-transformers
-scikit-learn
\ No newline at end of file
+transformers==4.28.1

```

Comment:
is it necessary to pin so specifically?

Reply:
probably not. I can allow it to update but at the version least I'll limit it by major version